client/src/components/ElementEditor/ElementActions.js

```diff
@@ -11,30 +11,31 @@ import AbstractAction from 'components/ElementActions/AbstractAction';
  * Element actions is a dropdown menu containing links to inline editing forms for each
  * of the element's primary tabs, as well as operations such as save, publish, archive etc
  */
-class ElementActions extends Component {
-  constructor(props) {
-    super(props);
-    this.handleEditTabsClick = this.handleEditTabsClick.bind(this);
-  }
-
+const ElementActions = ({
```

Comment:
The following props were added - previously referred to via this.props (in render()) though not in propTypes:
- id
- ActionMenuComponent
- children (though it's always a prop)